src/app/app.module.ts

```diff
@@ -131,6 +143,11 @@ import { CardComponent } from './card/card.component';
 
       { path: 'blog/:blog', component: BlogComponent, data: { state: 'base' }, canActivate: [LoginActivate] },
       { path: 'login', component: LoginComponent, data: { state: 'base' } },
+
+      { path: 'venter/complaints-home', component: ComplaintsHomeComponent, data: { state: 'base' } },
+      { path: 'venter/file-complaint', component: FileComplaintComponent, data: { state: 'base' } },
+      { path: 'venter/detailed-complaint/:id', component: DetailedComplaintComponent, data: { state: 'base' } },
```

Comment:
These paths are user-facing and hence should not be so verbose. Just change to:
1) `venter/complaints`
2) `venter/file`
3) `venter/complaint/:id`

Reply:
Also add `LoginActivate` here cause complaints aren't supposed to be visible to guests.